File: src/main/php/CreateExample.php
<?php
/*
 * This example shows how new databases can be created.
 *
 * Documentation: http://docs.basex.org/wiki/Clients
 *
 * (C) BaseX Team 2005-12, BSD License
 */
include("BaseXClient.php");

try {

  // create session
  $session = new Session("localhost", 1984, "admin", "admin");
  
  // create new database
  $session->create("database", "<x>Hello World!</x>");
  print $session->info();
  
  // run query on database
  print "<br/>".$session->execute("xquery /");
  
  // drop database
  $session->execute("drop db database");

  // close session
  $session->close();

} catch (Exception $e) {
  // print exception
  print $e->getMessage();
}
?>

File: src/main/ruby/CreateExample.rb
# This example shows how new databases can be created.
# Documentation: http://docs.basex.org/wiki/Clients
#
# (C) BaseX Team 2005-12, BSD License

require 'BaseXClient.rb'

begin

  # create session
  session = BaseXClient::Session.new("localhost", 1984, "admin", "admin")
  
  # create new database
  session.create("database", "<x>Hello World!</x>")
  print "\n" + session.info()
  
  # run query on database
  print "\n" + session.execute("xquery doc('database')") + "\n"
  
  # drop database
  session.execute("drop db database")

  # close session
  session.close

rescue Exception => e
  # print exception
  puts e
end
